cookbook/ifan02.rst

```diff
@@ -0,0 +1,181 @@
+Ifan02
+======
+
+.. seo::
+    :description: Instructions for using Sonoff ifan02 in ESPHome.
+    :keywords: Fan, Sonoff, ifan02
+
+
+Configuration for `Sonoff ifan02 <https://www.itead.cc/sonoff-ifan02-wifi-smart-ceiling-fan-with-light.html>`__.
```

Comment:
Longer description please. Explain what the iFan02 is, what it does and what functionality the user will get in the end.